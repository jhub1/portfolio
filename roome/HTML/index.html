<!DOCTYPE html>
<html lang="en">

<head>
    <meta charset="utf-8">
    <meta http-equiv="X-UA-Compatible" content="IE=edge">
    <meta name="viewport" content="width=device-width, initial-scale=1">
    <title>Roome</title>
    <link href="https://maxcdn.bootstrapcdn.com/bootstrap/3.3.7/css/bootstrap.min.css" rel="stylesheet">
    <link rel="stylesheet" href="../Assets/styles/animate.css">
    <link href="../Assets/styles/style.css" rel="stylesheet">
    <link rel="stylesheet" href="../Assets/plugins/fancybox/source/jquery.fancybox.css?v=2.1.6" type="text/css" media="screen" />
    <script src="https://use.typekit.net/vil7fgc.js"></script>
    <script>
    try {
        Typekit.load({
            async: true
        });
    } catch (e) {}
    </script>
</head>

<body>
    <section class="first-section">
        <div class="jumbotron">
            <div class="container">
                <div class="row">
                    <a href="#" class="col-xs-8 col-sm-6 col-md-3">
                        <img class="img-responsive" src="../Assets/images/logo.png" alt="Roome Logo"></a>
                    <a href="#" class="btn col-sm-3 col-md-3 pre-register pull-right hidden-sm hidden-xs">Pre-register</a>
                    <a href="#" class="col-xs-4 col-sm-6 col-md-2 col-md-offset-4"><img class="pull-right img-responsive" src="../Assets/images/indiegogo-logo.png" alt="Indiegogo Logo"></a>
                </div>
            </div>
        </div>
        <div class="container pre-register-image-container">
            <div class="row">
                <div class="col-xs-8 col-xs-offset-1 col-md-6 col-md-offset-1 heading">Light, anywhere you need</div>
            </div>
            <div class="row">
                <div class="col-xs-7 col-xs-offset-1 col-sm-6 subheading">Launching soon on <img src="../Assets/images/indiegogo-logo.png" alt="Indiegogo Logo"></div>
            </div>
            <!-- hide preregister button on smaller devices -->
            <div class="hidden-xs row pre-register-button-row">
                <a href="#" class="col-xs-offset-1 pre-register2">Pre-register</a>
            </div>
        </div>
        <!-- show preregister button on smaller devices -->
        <div class="container hidden-sm hidden-md hidden-lg">
            <div class="row">
                <a href="#" class="col-xs-10 col-xs-offset-1 pre-register2 mt1 text-center">Pre-register</a>
            </div>
        </div>
    </section>
    <section class="container second-section">
        <div class="row video-container">
            <div align="center" class="embed-responsive embed-responsive-16by9">
                <video controls class="embed-responsive-item">
                    <source src="../Assets/vid/roome-vid.mp4" type="video/mp4">
                </video>
            </div>
        </div>
    </section>
    <section class="third-section">
        <div class="jumbotron">
            <div class="container">
                <div class="row">
                    <div class="col-xs-12 col-md-6 roome-image">
                        <img src="../Assets/images/a1.png" class="img-responsive" alt="Roome Image"></a>
                    </div>
                    <div class="col-md-6 description">
                        <h1>Light up your imagination</h1>
                        <p>The RoomeLight Mini+ is an amazing smart lamp which learns your behavior using the latest in presence detection technology. RoomeLight Mini+ is also a perfect portable lamp for outdoor use. With only 2 hours of charging, it can provide up to 10 hours of continuous lighting.</p>
                    </div>
                </div>
            </div>
        </div>
    </section>
    <section class="container fourth-section">
        <div class="row features">
            <!-- Hide this on small devices -->
            <div class="hidden-xs hidden-sm col-sm-6 col-md-4 text-right left-column">
                <div class="row feature-row">
                    <div class="col-md-7 feature-description">dimmable 150 lumen</div>
                    <div class="col-md-5">
                        <img class="feature-image" src="../Assets/images/dim.jpg" alt="Presence Detection"></a>
                    </div>
                </div>
                <div class="row feature-row">
                    <div class="col-md-7 feature-description">comfortable led light</div>
                    <div class="col-md-5">
                        <img class="feature-image" src="../Assets/images/comfort.jpg" alt="Presence Detection"></a>
                    </div>
                </div>
                <div class="row feature-row">
                    <div class="col-md-7 feature-description">micro usb charging</div>
                    <div class="col-md-5">
                        <img class="feature-image" src="../Assets/images/micro.jpg" alt="Presence Detection"></a>
                    </div>
                </div>
            </div>
            <div class="hidden-xs hidden-sm col-md-4 text-center">
                <img src="../Assets/images/roome-image.png" alt="Roome Image"></a>
            </div>
            <div class="hidden-xs hidden-sm col-sm-6 col-md-4 text-left right-column">
                <div class="row feature-row">
                    <div class="col-md-5">
                        <img class="feature-image" src="../Assets/images/app.jpg" alt="Presence Detection"></a>
                    </div>
                    <div class="col-md-7 feature-description">app control</div>
                </div>
                <div class="row feature-row">
                    <div class="col-md-5">
                        <img class="feature-image" src="../Assets/images/smart.jpg" alt="Presence Detection">
                    </div>
                    <div class="col-md-7 feature-description">Smart Learning</div>
                </div>
                <div class="row feature-row">
                    <div class="col-md-5">
                        <img class="feature-image" src="../Assets/images/detect.jpg" alt="Presence Detection">
                    </div>
                    <div class="col-md-7 feature-description">Presence Detection</div>
                </div>
            </div>
            <!-- Show this on small devices -->
            <div class="hidden-md hidden-lg col-xs-12">
                <div class="row feature-row">
                    <div class="col-xs-6 text-center">
                        <img class="feature-image" src="../Assets/images/dim.jpg" alt="Presence Detection">
                        <div class="feature-description">dimmable 150 lumen</div>
                    </div>
                    <div class="col-xs-6 text-center">
                        <img class="feature-image" src="../Assets/images/comfort.jpg" alt="Presence Detection">
                        <div class="feature-description">comfortable led light</div>
                    </div>
                </div>
                <div class="row feature-row">
                    <div class="col-xs-6 text-center">
                        <img class="feature-image" src="../Assets/images/micro.jpg" alt="Presence Detection">
                        <div class="feature-description">micro usb charging</div>
                    </div>
                    <div class="col-xs-6 text-center">
                        <img class="feature-image" src="../Assets/images/app.jpg" alt="Presence Detection"></a>
                        <div class="feature-description">app control</div>
                    </div>
                </div>
                <div class="row feature-row">
                    <div class="col-xs-6 text-center">
                        <img class="feature-image" src="../Assets/images/smart.jpg" alt="Presence Detection">
                        <div class="feature-description">Smart Learning</div>
                    </div>
                    <div class="col-xs-6 text-center">
                        <img class="feature-image" src="../Assets/images/detect.jpg" alt="Presence Detection">
                        <div class="feature-description">Presence Detection</div>
                    </div>
                </div>
            </div>
            <div class="hidden-md hidden-lg col-xs-12 text-center">
                <img src="../Assets/images/roome-image.png" alt="Roome Image"></a>
            </div>
        </div>
        <div class="row buttons-row text-center">
            <a href="#" class="btn btn-lg col-xs-10 col-xs-offset-1 col-sm-4 col-sm-offset-1 col-md-4 col-md-offset-3 more-info">More information</a>
            <a href="#" class="btn btn-lg col-xs-10 col-xs-offset-1 col-sm-4 col-md-3 col-md-offset-0 pre-register">Pre-register</a>
        </div>
    </section>
    <section class="fifth-section">
        <div class="container">
            <div class="row">
                <div class="col-xs-4 usage">
                    <a class="fancybox" rel="group" href="../Assets/images/usage-1.jpg"><img class="img-responsive" src="../Assets/images/usage-1.jpg" alt="Roome Image"></a>
                </div>
                <div class="col-xs-4 usage">
                    <a class="fancybox" rel="group" href="../Assets/images/usage-2.jpg"><img class="img-responsive" src="../Assets/images/usage-2.jpg" alt="Roome Image"></a>
                </div>
                <div class="col-xs-4 usage">
                    <a class="fancybox" rel="group" href="../Assets/images/usage-3.jpg"><img class="img-responsive" src="../Assets/images/usage-3.jpg" alt="Roome Image"></a>
                </div>
            </div>
            <div class="row">
                <div class="col-xs-4 usage">
                    <a class="fancybox" rel="group" href="../Assets/images/usage-4.jpg"><img class="img-responsive" src="../Assets/images/usage-4.jpg" alt="Roome Image"></a>
                </div>
                <div class="col-xs-4 usage">
                    <a class="fancybox" rel="group" href="../Assets/images/usage-5.jpg"><img class="img-responsive" src="../Assets/images/usage-5.jpg" alt="Roome Image"></a>
                </div>
                <div class="col-xs-4 usage">
                    <a class="fancybox" rel="group" href="../Assets/images/usage-6.jpg"><img class="img-responsive" src="../Assets/images/usage-6.jpg" alt="Roome Image"></a>
                </div>
            </div>
        </div>
    </section>
    <section class="sixth-section">
        <div class="jumbotron">
            <div class="container subscribe-container">
                <div class="row">
                    <div class="subscribe-heading col-xs-8 col-xs-offset-2 col-sm-8 col-sm-offset-1">Sign up to get notified when we launch our indiegogo campaign</div>
                </div>
                <div class="row">
                    <div class="col-xs-8 col-xs-offset-2 col-sm-10 col-sm-offset-1  col-md-8 col-md-offset-1 email-input">
                        <div class="form-group">
                            <input type="email" class="form-control input-lg" id="inputEmail" placeholder="Email Address">
                        </div>
                    </div>
                    <a href="#" class="btn btn-lg col-xs-8 col-xs-offset-2 col-sm-10 col-sm-offset-1 col-md-2 col-md-offset-0 subscribe-button">subscribe</a>
                </div>
            </div>
        </div>
    </section>
    <section class="seventh-section">
        <div class="jumbotron">
            <div class="container footer-container">
                <div class="row">
                    <div class="col-xs-12 col-sm-6 footer-logo">
                        <img src="../Assets/images/logo-blue.png" alt="Roome Logo">
                    </div>
                    <div class="col-xs-12 col-sm-6 share-logos">
                        <a href="#"><img src="../Assets/images/twitter.png" alt="Twitter Logo"></a>
                        <a href="#"><img src="../Assets/images/facebook.png" alt="Facebook Logo"></a>
                        <a href="#"><img src="../Assets/images/instagram.png" alt="Instagram Logo"></a>
                    </div>
                </div>
            </div>
        </div>
    </section>
    <script src="https://ajax.googleapis.com/ajax/libs/jquery/1.12.4/jquery.min.js"></script>
    <script type="text/javascript" src="../Assets/plugins/fancybox/source/jquery.fancybox.pack.js?v=2.1.6"></script>
    <script src="https://maxcdn.bootstrapcdn.com/bootstrap/3.3.7/js/bootstrap.min.js"></script>
    <script src="../Assets/js/index.js"></script>
</body>

</html>
