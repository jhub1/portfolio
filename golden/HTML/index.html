<!DOCTYPE html>
<html lang="en">

<head>
    <meta charset="utf-8">
    <meta http-equiv="X-UA-Compatible" content="IE=edge">
    <meta name="viewport" content="width=device-width, initial-scale=1">
    <title>Golden</title>
    <link href="https://maxcdn.bootstrapcdn.com/bootstrap/3.3.7/css/bootstrap.min.css" rel="stylesheet">
    <link href="../Assets/styles/animate.css" rel="stylesheet">
    <link href="../Assets/styles/style.css" rel="stylesheet">
    <link href="https://fonts.googleapis.com/css?family=Droid+Serif:400i|Montserrat:700|Roboto+Slab" rel="stylesheet">
</head>

<body>
    <section class="first-section-wrapper">
        <nav class="navbar navbar-default navbar-custom ">
            <div class="container">
                <div class="navbar-header">
                    <button type="button" class="navbar-toggle collapsed" data-toggle="collapse" data-target="#header-navbar-collapse" aria-expanded="false">
                        <span class="sr-only">Toggle navigation</span>
                        <span class="icon-bar"></span>
                        <span class="icon-bar"></span>
                        <span class="icon-bar"></span>
                    </button>
                    <a class="navbar-brand" href="#"><img src="../Assets/images/logo.png" alt="Golden Logo"></a>
                </div>
                <div class="collapse navbar-collapse" id="header-navbar-collapse">
                    <ul class="nav navbar-nav navbar-right">
                        <li><a href="#">Home</a></li>
                        <li><a href="#">Services</a></li>
                        <li><a href="#">Portfolio</a></li>
                        <li><a href="#">About</a></li>
                        <li><a href="#">Contact</a></li>
                    </ul>
                </div>
            </div>
        </nav>
        <div class="container home">
            <div class="row home-content text-center">
                <div class="col-xs-12 welcome-msg-1">Welcome to our studio!</div>
                <div class="col-xs-12 welcome-msg-2">It's nice to meet you</div>
                <a href="#" class="home-button">Tell me more</a>
            </div>
        </div>
    </section>
    <section class="container services-container text-center">
        <div class="services-content">
            <div class="row">
                <div class="title">Services</div>
                <div class="sub-title">Lorem ipsum dolorem quid quo</div>
            </div>
            <div class="row services">
                <div class="col-xs-12 col-sm-6 col-md-4 animated fadeInUp">
                    <img src="../Assets/images/ecommerce-icon.png" />
                    <h4 class="name">E-Commerce</h4>
                    <p class="description">Lorem ipsum dolor sit amet, consectetur adipiscing elit. Mauris sed diam ac lectus ullamcorper elementum. Aenean eu pellentesque libero. Integer urna nunc, ultricies a velit quis, feugiat eleifend arcu.</p>
                </div>
                <div class="col-xs-12 col-sm-6 col-md-4 animated fadeInUp">
                    <img src="../Assets/images/responsive-web-icon.png" />
                    <h4 class="name">Responsive Web</h4>
                    <p class="description">Vestibulum ante ipsum primis in faucibus orci luctus et ultrices posuere cubilia Curae; Duis nec nunc porttitor odio vestibulum porttitor at vitae massa. </p>
                </div>
                <div class="col-xs-12 col-sm-6 col-md-4 animated fadeInUp">
                    <img src="../Assets/images/web-security-icon.png" />
                    <h4 class="name">Web Security</h4>
                    <p class="description">Phasellus fermentum augue sit amet sapien lobortis interdum. Nunc eu leo et tellus tempor interdum. Sed sagittis tortor sit amet dui accumsan, vitae vehicula felis scelerisque.</p>
                </div>
            </div>
        </div>
    </section>
    <section class="jumbotron portfolio-container text-center">
        </div>
        <div class="container">
            <div class="portfolio-content">
                <div class="row">
                    <div class="title">Portfolio</div>
                    <div class="sub-title">Lorem ipsum dolorem quid quo</div>
                </div>
                <div class="row portfolio">
                    <div class="col-xs-12 col-sm-6 col-md-4">
                        <img src="../Assets/images/portfolio-1.png" />
                        <div class="name-description">
                            <h4 class="name">Ebony & Ivory</h4>
                            <p class="description">Branding</p>
                        </div>
                    </div>
                    <div class="col-xs-12 col-sm-6 col-md-4">
                        <img src="../Assets/images/portfolio-2.png" />
                        <div class="name-description">
                            <h4 class="name">Smart Stationary</h4>
                            <p class="description">Print Design</p>
                        </div>
                    </div>
                    <div class="col-xs-12 col-sm-6 col-md-4">
                        <img src="../Assets/images/portfolio-3.png" />
                        <div class="name-description">
                            <h4 class="name">Clever Poster</h4>
                            <p class="description">Print Design</p>
                        </div>
                    </div>
                    <div class="col-xs-12 col-sm-6 col-md-4">
                        <img src="../Assets/images/portfolio-4.png" />
                        <div class="name-description">
                            <h4 class="name">Vinyl Record</h4>
                            <p class="description">Product Mock-Up</p>
                        </div>
                    </div>
                    <div class="col-xs-12 col-sm-6 col-md-4">
                        <img src="../Assets/images/portfolio-5.png" />
                        <div class="name-description">
                            <h4 class="name">Treehouse Template</h4>
                            <p class="description">Web Design</p>
                        </div>
                    </div>
                    <div class="col-xs-12 col-sm-6 col-md-4">
                        <img src="../Assets/images/portfolio-6.png" />
                        <div class="name-description">
                            <h4 class="name">Burned Logo</h4>
                            <p class="description">Branding</p>
                        </div>
                    </div>
                </div>
            </div>
    </section>
    <section class="about-us-container text-center">
        <div class="container about-us-content">
            <div class="row">
                <div class="title">About Us</div>
                <div class="sub-title">Lorem ipsum dolorem quid quo</div>
            </div>
            <div class="container about-us">
                <div class="row odd">
                    <img class="odd-row-image" src="../Assets/images/about-us-1.jpg" />
                    <div class="col-xs-7 col-xs-offset-5 col-sm-4 col-sm-offset-0  col-md-4 col-lg-4 col-lg-offset-1 timeline-text">
                        <h4>July 2010</h4>
                        <h4>Our Humble Begginings</h4>
                        <div class="about-us-desc">Proin iaculis purus consequat sem cure digni ssim. Donec porttitora entum suscipit aenean rhoncus posuere odio in tincidunt. Proin iaculis purus consequat sem cure digni ssim. Donec porttitora entum suscipit.</div>
                    </div>
                </div>
                <div class="row even">
                    <img class="even-row-image" src="../Assets/images/about-us-2.jpg" />
                    <div class="col-xs-7 col-xs-offset-5 col-sm-4 col-sm-offset-8 col-md-4 col-md-offset-8 col-lg-4 col-lg-offset-7 timeline-text">
                        <h4>January 2011</h4>
                        <h4>Facing Startup Battles</h4>
                        <div class="about-us-desc">Proin iaculis purus consequat sem cure digni ssim. Donec porttitora entum suscipit aenean rhoncus posuere odio in tincidunt. Proin iaculis purus consequat sem cure digni ssim. Donec porttitora entum suscipit aenean rhoncus posuere odio in tincidunt.</div>
                    </div>
                </div>
                <div class="row odd">
                    <img class="odd-row-image" src="../Assets/images/about-us-3.jpg" />
                    <div class="col-xs-7 col-xs-offset-5 col-sm-4 col-sm-offset-0  col-md-4 col-lg-4 col-lg-offset-1 timeline-text">
                        <h4>December 2012</h4>
                        <h4>Enter The Dark Days</h4>
                        <div class="about-us-desc">Proin iaculis purus consequat sem cure digni ssim. Donec porttitora entum suscipit aenean rhoncus posuere odio in tincidunt. Proin iaculis purus consequat sem cure digni.</div>
                    </div>
                </div>
                <div class="row even">
                    <img class="even-row-image" src="../Assets/images/about-us-4.jpg" />
                    <div class="col-xs-7 col-xs-offset-5 col-sm-4 col-sm-offset-8 col-md-4 col-md-offset-8 col-lg-4 col-lg-offset-7 timeline-text">
                        <h4>February 2014</h4>
                        <h4>Our Triumph</h4>
                        <div class="about-us-desc">Proin iaculis purus consequat sem cure digni ssim. Donec porttitora entum suscipit aenean rhoncus posuere odio in tincidunt. Proin iaculis purus consequat sem cure digni ssim. Donec porttitora entum suscipit aenean.</div>
                    </div>
                </div>
                <div class="row odd">
                    <div class="odd-row-image last">Our Story Continues ...</div>
                </div>
            </div>
        </div>
    </section>
    <section class="our-team-container text-center">
        <div class="container our-team-content">
            <div class="row">
                <div class="title">Our Amazing Team</div>
                <div class="sub-title">Lorem ipsum dolorem quid quo</div>
            </div>
            <div class="container our-team">
                <div class="row">
                    <div class="col-xs-12 col-sm-6 col-md-4">
                        <img src="../Assets/images/our-team-3.jpg">
                        <h4>Kimberly Thompson</h4>
                        <div class="role">Marketer</div>
                        <div>
                            <div class="twitter"></div>
                            <div class="facebook"></div>
                            <div class="pinterest"></div>
                        </div>
                    </div>
                    <div class="col-xs-12 col-sm-6 col-md-4">
                        <img src="../Assets/images/our-team-2.jpg">
                        <h4>Rico Massimo</h4>
                        <div class="role">Coder</div>
                        <div>
                            <div class="twitter"></div>
                            <div class="facebook"></div>
                            <div class="pinterest"></div>
                        </div>
                    </div>
                    <div class="col-xs-12 col-sm-6 col-md-4">
                        <img src="../Assets/images/our-team-1.jpg">
                        <h4>Uku Mason</h4>
                        <div class="role">Graphic Designer</div>
                        <div>
                            <div class="facebook"></div>
                            <div class="pinterest"></div>
                        </div>
                    </div>
                </div>
                <div class="row">
                    <div class="col-xs-12 col-md-6 col-md-offset-3 our-team-bottom-text">Proin iaculis purus consequat sem cure digni ssim donec porttitora entum suscipit aenean rhoncus posuere odio in tincidunt proin iaculis.</div>
                </div>
            </div>
        </div>
    </section>
    <section class="clients-container text-center">
        <div class="container clients-content">
            <div class="row clients">
                <div class="col-xs-6 col-sm-6 col-md-3">
                    <img src="../Assets/images/envato.png" alt="">
                </div>
                <div class="col-xs-6 col-sm-6 col-md-3">
                    <img src="../Assets/images/wordpress.png" alt="">
                </div>
                <div class="col-xs-6 col-sm-6 col-md-3">
                    <img src="../Assets/images/tuts.png" alt="">
                </div>
                <div class="col-xs-6 col-sm-6 col-md-3">
                    <img src="../Assets/images/microlancer.png" alt="">
                </div>
            </div>
        </div>
    </section>
    <section class="contact-us-container text-center">
        <div class="container contact-us-content">
            <div class="row">
                <div class="title">Our Amazing Team</div>
                <div class="sub-title">Lorem ipsum dolorem quid quo</div>
            </div>
            <div class="row contact-us">
                <div class="col-xs-12 col-sm-6 col-md-5 col-md-offset-1">
                    <div class="form-group">
                        <input type="email" class="form-control" id="inputEmail" placeholder="EMAIL *">
                    </div>
                    <div class="form-group">
                        <input type="password" class="form-control" id="inputPassword" placeholder="PASSWORD *">
                    </div>
                    <div class="form-group">
                        <input type="text" class="form-control" id="inputSubject" placeholder="SUBJECT">
                    </div>
                </div>
                <div class="col-xs-12 col-sm-6 col-md-5">
                    <div class="form-group">
                        <textarea rows="9" class="form-control" id="inputMessage" placeholder="YOUR MESSAGE *"></textarea>
                    </div>
                </div>
                <a href="#" class="col-xs-10 col-xs-offset-1 col-sm-4 col-sm-offset-4 col-md-2 col-md-offset-5 contact-us-button">Send Message</a>
            </div>
        </div>
    </section>
    <section class="footer-container">
        <div class="container footer-content">
            <div class="row footer">
                <div class="col-xs-12 col-sm-6 col-md-4 copyright">
                    <div class="row">
                        <div class="col-xs-12">© Copyright 2014 FreebiesXpress.com</div>
                        <div class="col-xs-12">© Copyright 2017 Implemented by Marko Simundza</div>

                    </div>
                </div>
                <div class="col-xs-12 col-sm-6 col-md-4 text-center share-icons">
                    <div class="twitter-f"></div>
                    <div class="facebook-f"></div>
                    <div class="pinterest-f"></div>
                    <div class="google-f"></div>
                </div>
            </div>
    </section>
    <script src="https://ajax.googleapis.com/ajax/libs/jquery/1.12.4/jquery.min.js "></script>
    <script src="https://maxcdn.bootstrapcdn.com/bootstrap/3.3.7/js/bootstrap.min.js "></script>
</body>

</html>
